### PR TITLE
First pass at comprehensive new table of contents

### PR DESCRIPTION
**Pull request by jhodapp**
This commit brings a first round of comprehensive changes to our Core documentation table of contents (ToC).

### CODE REVIEW

**Comment by knitzsche** [comment]
It is worth noting perhaps that this represents a team effort developed here: https://docs.google.com/spreadsheets/d/1T6CLx--gZ7csa7jR84HFnsrQkUmTpnCv2beliG5ZPmQ/edit?ts=5a297c77#gid=673678892

File: docs/en/metadata.yaml [hunk 1]
```diff
@@ -1,47 +1,228 @@
...
+          - title: Steps to build an image
+            location: steps-build-image.md
+
+          - title: Post image build customization
+            location: post-image-build-customization.md
+
+          - title: Flashing image
+            location: flashing-image.md
+
+          - title: Flashing image
```
Comment (caldav) [inline docs/en/metadata.yaml]: Duplicated item

Comment (evilnick) [inline docs/en/metadata.yaml]: Repeat of above

**Comment by caldav** [comment]
Comments left on the spreadhseet, so we can sort things out there first. Mostly about renaming sections.

**Comment by jhodapp** [comment]
Here's a link to a latest build of the current documentation. I'll try and keep this up to date as I push changes so it's easy for everyone to preview what it will actually look like in a web browser.

https://drive.google.com/file/d/1Dw2SEOzITjW2e4co4xjnWmfPg0TibHcr/view?usp=sharing

**Pull request merged**